### ISSUE TITLE
Virtual keyboard focus out issue upon keypress.

### ISSUE THREAD

**Issue by arjunexalture** [arnog/mathlive#2240]
I am using mathlive virtual keyboard for touch devices. It works as expected in browser. 
When I use it in our app, what happens is after a keypress from virtual keyboard, data updates and then text editor getting focused out. Upon next keypress, the virtual keyboard disappears. Is there anything to prevent from this focus out situation.

**Comment by eliu-fracta** [comment]
I got the same issue

https://github.com/arnog/mathlive/issues/2182

**Comment by edward1127** [comment]
I got the same issue

https://github.com/arnog/mathlive/issues/2182

**Comment by arnog** [comment]
@edward1127 is it the same issue? I thought your issue was triggered when typing quickly? That doesn't seem to be the case here. Have you been able to find a resolution or workaround for your issue?

@arjunexalture could you provide more details? Can you build a reproducible test case that could be debugged? Are you using React, React Native or Angular?

**Comment by edward1127** [comment]
@arnog Yep, that's correct, but I think it might be related since it also loses focus, and it only happens in nextjs but not react. I'm still checking which line of the code might be the culprit. 
https://github.com/vercel/next.js/discussions/60076

**Comment by arjunexalture** [comment]
@arnog I am using React and I am using sandbox mode inside iframe. In my case, it's not reproduceable in browser. It is happening in app version of my project only.

**Comment by arnog** [comment]
@arjunexalture can you provide any additional information that could help narrow down this issue? Do you have some code you could share? Or a stack trace for when the blur event is handled?

**Comment by arnog** [comment]
Some improvements to the handling of VK inside iframe have been made and may have improved this issue.
